plugins/woocommerce/changelog/46626-fix-eslint-errors

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+Comment: Fix ESLint errors
```

Comment:
Not a big deal, but I think this PR does much more than this. :smile: It might be good to update the comment to align with the PR contents.

Reply:
Haha definitely! That must have been merged along with #46626 that fixed the linting errors. Will update!